### DISCUSSION TITLE
Any thoughts on using table to show parent->children?

### DISCUSSION THREAD

**Discussion by mallanza** [rappasoft/laravel-livewire-tables#1639] [Q&A]
Wondering if I can apply table to my hierarchy, instead of writing my own layouts? Checking for SME opinions

![image](https://github.com/rappasoft/laravel-livewire-tables/assets/153227702/dbaa7400-93fb-4ed9-8cb3-543327e928e1)

**Comment by lrljoe** [comment]
What do you mean by applying a table to the hierarchy, do you mean displaying a Column with that format?

**Reply by mallanza** [reply]
I am talking about children
```
    public function children(): HasMany
    {
        return $this->hasMany(Organization::class, 'parent_id', 'id');
    }
```
When I show the table, I want to be able to have "grouping' with children or come up with some clever way to show it (as I have on the image above)

(an example https://www.codeply.com/p/Aju4qar5sx)

**Reply by lrljoe** [reply]
The short answer is, you can use a custom view for the Column, or use a label method to display it

I am planning to add in the approach I use for doing similar in the near future as a custom column

Feel free to reach out on the Discord if you need an interim pointer
